cmd/merge.go

```diff
@@ -22,6 +22,8 @@ func mergeInit() {
 	_ = mergeCmd.MarkFlagRequired("schemas")
 	mergeCmd.Flags().StringP("out", "o", "", "path to the output file")
 	_ = mergeCmd.MarkFlagRequired("out")
+	mergeCmd.Flags().String("base-path", "", "optional base path to resolve relative $ref file references")
```

Comment:
Is this argument necessary? Did you consider deriving the base-path from the `filepath.Dir` of the OAS document?

Reply:
I did not! but I'm also wondering if there's going to be a case where the referenced components in the root doc live in a different folder -- it's a bit strange but I noticed some tools do ask for a basePath. 

I could set `filepath.Dir` as a default and override if they choose and make `base-path` optional?

Reply:
Personally would find that case confusing. The nice thing about the OAS dir being the base path is that that's the IDE assumption, so most of them with json point support would already allow navigation through the document tree by ctrl-clicking. We also make this assumption throughout the CLI and don't make the base path configurable.

We should always pass it in though to libopenapi as otherwise the base path is implicitly the current working directory, so would see errors if the CLI is ever pointed at a spec in a different folder.

Reply:
okay makes sense! Will update and remove the option then.